crates/provider/src/heart.rs

```diff
@@ -108,18 +108,18 @@ impl<'a, N: Network, T: Transport + Clone> PendingTransactionBuilder<'a, N, T> {
     }
 
     /// Returns the number of confirmations to wait for.
-    pub const fn confirmations(&self) -> u64 {
-        self.config.confirmations()
+    pub const fn reqd_confs(&self) -> u64 {
```

Comment:
maybe `#[doc(alias)]` on at least one of these for discoverability?

Reply:
I'll go ahead and alias them all 👍